_data/applications.yml

```diff
@@ -0,0 +1,175 @@
+  package: cockpit-kdump
+  description: |
+    Debugging the Linux kernel? Cockpit's kdump add-on helps with catching stack traces.
+
+networkmanager:
+  title: Network
+  official: true
+  source: cockpit
+  package: cockpit-networkmanager
+  description: |
+    Manage your network interfaces and edit your firewall with ease.
+
+packagekit:
+  title: Software Updates (PackageKit)
+  official: true
+  source: cockpit
+  package: cockpit-packagekit
+  distro: Fedora, Red Hat Enterprise Linux, Debian, Ubuntu
+  description: |
+    See and apply updates to your system. Supports RPM and DEB based systems through PackageKit.
+
+ostree:
+  title: Software Updates (OSTree)
+  official: true
+  package: cockpit-ostree
+  source: https://github.com/cockpit-project/cockpit-ostree
+  issues: https://github.com/cockpit-project/cockpit-ostree/issues
+  license: LGPL 2.1
+  description: |
+    Upgrade OSTree-based systems in your browser.
+
+composer:
+  title: Cockpit Composer
+  package: cockpit-composer
+  source: https://github.com/osbuild/cockpit-composer/
+  issues: https://github.com/osbuild/cockpit-composer/issues
+  description: |
+     Composer is an interface for OSBuild that generates custom images suitable for deploying systems or uploading to the cloud. Images built with Composer are for RPM-based distributions such as Fedora, Red Hat Enterprise Linux, and CentOS.
+
+certificates:
+  title: Certificate Management
+  package: cockpit-certificates
+  source: https://github.com/skobyda/cockpit-certificates
+  issues: https://github.com/skobyda/cockpit-certificates/issues
+  prerelease: true
+  description: |
+    Certificate management app for Cockpit.
+
+sosreport:
+  title: Diagnostic Reports
+  official: true
+  package: cockpit-sosreport
+  source: cockpit
+  description: |
+    A tool to collect system configuration and diagnostic information from systems, to help with diagnosing problems.
+
+389-ds:
+  title: 389 Directory Server
+  package: cockpit-389-ds
+  homepage: https://directory.fedoraproject.org/docs/389ds/download.html
+  source: 
+  description: |
+    A web-based interface to the enterprise-class Open Source LDAP server for Linux.
+
+session-recording:
+  package: cockpit-session-recording
+  title: Session Recording
+  source: https://github.com/Scribery/cockpit-session-recording
+  issues: https://github.com/Scribery/cockpit-session-recording/issues
+  description: |
+    Session recording configuration and playback for Cockpit, based on [Scribery](https://scribery.github.io/), to record a user's actions for watching later.
+
+subscriptions:
+  package: cockpit-subscriptions
+  title: Subscription Manager
+  source: https://github.com/candlepin/subscription-manager
+  issues: 
+  description: |
+    Manage subscriptions to Red Hat Enterprise Linux from your web browser.
+
+leapp:
+  package: cockpit-leap
+  title: Leapp
+  homepage: https://leapp.readthedocs.io/en/latest/
+  source: https://github.com/oamg/leapp
```

Comment:
Somehow this does not appear on the rendered page.

Reply:
It has to have a title and description, I think. Perhaps I forgot the description, although I thought I wrote it. I'll re-check.

Reply:
Oh you mean source. That used to be there. It was asked to be removed. But that's before I changed the layout.